components/tag/TagCard.vue

```diff
@@ -11,12 +11,34 @@ const to = $computed(() => {
   const { hostname, pathname } = new URL(tag.url)
   return `/${hostname}${pathname}`
 })
+
+const router = useRouter()
+
+function onclick(evt: MouseEvent | KeyboardEvent) {
+  const path = evt.composedPath() as HTMLElement[]
+  const el = path.find(el => ['A', 'BUTTON'].includes(el.tagName?.toUpperCase()))
+  const text = window.getSelection()?.toString()
+  if (!el && !text)
+    go(evt)
+}
+
+function go(evt: MouseEvent | KeyboardEvent) {
+  if (evt.metaKey || evt.ctrlKey)
+    window.open(to)
+  else
+    router.push(to)
+}
```

Comment:
I would be more comfortable if we were able to use a normal link. can we prevent the click from bubbling up from the button instead?

Reply:
I tried, and nope

This is copied and adjusted from https://github.com/elk-zone/elk/blob/2a836eda3d2db3cb37589b7ea97f9c23e6af0c58/components/status/StatusCard.vue#L46-L64

Later (in a separate PR) we could extract it into a composable

I tried: `@click.stop`, `@click.prevent`, `@click.native` and some other things
Then I looks how the actions in a status are doing it, and found this above...